e2e-tests/tests.sh

```diff
@@ -23,7 +23,8 @@ finish() {
 trap finish EXIT
 
 kubectl create -f manifests/webhook.yaml
-awk '/image: / { print; print "        imagePullPolicy: Never"; next }1' manifests/cert-manager-job.yaml | kubectl create -f -
+#awk '/image: / { print; print "        imagePullPolicy: Never"; next }1' manifests/cert-manager-job.yaml | 
+kubectl create -f manifests/cert-manager-job.yaml
```

Comment:
I understand this is not necessary if `imagePullPolicy: IfNotPresent` is set above, but I'm curious about the motivation for this change. Is there any benefit over the previous solution? If so, I can implement it on my pipeline as well.

Reply:
The awk command didn't work for me. It still complained that the imagepullPolicy was "wrong" and wouldn't deploy. 
The way I found to solve it was to set it directly in the yaml. Don't know for sure why we need "awk" command here since  we're only using this yaml here and we're always doing it, so why not set it directly in the yaml? Maybe I'm missing something..

Reply:
Having the imagePullPolicy set to `IfNotPresent` do not we risk that it downloads the latest from dockerhub instead of the built one?

Reply:
>  The way I found to solve it was to set it directly in the yaml. Don't know for sure why we need "awk" command here since we're only using this yaml here and we're always doing it, so why not set it directly in the yaml? Maybe I'm missing something.

I checked and I do not see the yaml being used anywhere else either. Perhaps this is a copy-paste from other pipelines (such as https://github.com/newrelic/k8s-metadata-injection) where the same yaml is actually pushed to prod, so the awk is needed. Since it is not the case here I agree that changing the yaml directly is better :)

I would however try with `Never` rather than `IfNotPresent` just to be in the safe side, as Paolo says. It would be hard to notice if for some reason the image being tested starts being the remote one.